lib/services/init-service.ts

```diff
@@ -56,6 +57,12 @@ export class InitService implements IInitService {
 					});
 				}
 
+				let dependencies = projectData["dependencies"];
+				if(!dependencies) {
+					projectData["dependencies"] = Object.create(null);
+				}
+				projectData["dependencies"][constants.TNS_CORE_MODULES_NAME] = this.getVersionData(constants.TNS_CORE_MODULES_NAME).wait()["version"];
```

Comment:
Perhaps `projectData.dependencies[constants.TNS_CORE_MODULES_NAME] = this.getVersionData(constants.TNS_CORE_MODULES_NAME).wait().version`